packages/rancher-cis-benchmark/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cis-operator
+  namespace: {{ template "cis.namespace" . }}
+  labels:
+    cis.cattle.io/operator: cis-operator
+spec:
+  selector:
+    matchLabels:
+      cis.cattle.io/operator: cis-operator
+  template:
+    metadata:
+      labels:
+        cis.cattle.io/operator: cis-operator
+    spec:
+      serviceAccountName: cis-operator-serviceaccount
+      containers:
+      - name: cis-operator
+        image: '{{ template "system_default_registry" . }}{{ .Values.image.cisoperator.repository }}:{{ .Values.image.cisoperator.tag }}'
+        imagePullPolicy: Always
```

Comment:
Any reason to use Always and not `IfNotPresent` ?

Reply:
Wont `Always` ensure that the latest image is pulled always even when chart is upgraded? Also actually I just carried forward this from the CIS v1 rancher-cis-benchmark system-charts https://github.com/rancher/system-charts/blob/dev-v2.4/charts/rancher-cis-benchmark/0.1.2/values.yaml#L31

Reply:
When the chart is upgraded the version tag should change and new image would get pulled. But looking into it, `always` just does a check against cached image, so this should be fine.